.eslintrc.json

```diff
@@ -0,0 +1,14 @@
+{
+    "env": {
+        "commonjs": true,
+        "es2021": true,
+        "node": true
+    },
+    "extends": "eslint:recommended",
```

Comment:
Let's use [Airbnb's](https://medium.com/@Tunmise/set-up-eslint-with-airbnb-style-guide-in-5-minutes-d7b4cc5707f8) eslint style guide